src/scripts/project/remove-files-by-pattern.ts

```diff
@@ -3,18 +3,20 @@
  *
  * Parameters:
  *  --searchDir Directory where script will do search
+ *
```

Comment:
There is a strange thing with `@type {string}`. Maybe we should remove such line and add type near parameter definition.

```
@param {string} searchDir Directory where script will do search
@param {string} pattern RegExp pattern without slashes (/). For example, styles.d.ts$ becomes /styles.d.ts$/

@example
    
    bundle-dts.ts --searchDir ./some/dir --pattern styles.d.ts$
```

Link to jsdoc cheetsheet: https://devhints.io/jsdoc